test/cpp/jit/test_backend.cpp

```diff
@@ -1,81 +1,42 @@
+#include <gtest/gtest.h>
+#include <torch/csrc/jit/api/module.h>
 #include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_detail.h>
+#include <torch/csrc/jit/frontend/resolver.h>
+#include <torch/torch.h>
```

Comment:
Why is this header needed? I'm not familiar with it so it might be used, I'm just curious.

Reply:
It's a header of all file, probably for ease of use, especially for generated headers? The tensor factory methods (torch::ones) are from what this header includes.